frontend/src/components/dashboard/create/email/EmailTemplate.tsx

```diff
@@ -70,13 +72,19 @@ const EmailTemplate = ({
         value={subject}
         onChange={setSubject}
       />
-      <h4>Message</h4>
+      <h4>{protect ? 'Message A' : 'Message'}</h4>
```

Comment:
Message A may make no sense to the user. Will prefer something like Plaintext message, or something.. TBD